### DISCUSSION TITLE
Return ALL_OLD on delete request

### DISCUSSION THREAD

**Discussion by onhate** [sensedeep/dynamodb-onetable#266] [Ideas]
Hello there, opening a discussion about returning ALL_OLD attributes when removing a record, I'll work on a PR for this

https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/DynamoDB_20111205.html#deleteItem-property

**Comment by mobsense** [comment]
That sounds reasonable. Need to also handle a return value if the item does not exist.

The code for this is in Expression.js in the command() function. You'll see code that handles 'put' and 'update' return values. A similar section for delete would do the trick.

**Comment by onhate** [comment]
#291
